.gitignore

```diff
@@ -3,9 +3,7 @@ notebooks/
 example/
 .DS_Store
 .env
-src/**/*.json
-results
-./*.json
+
```

Comment:
please do not change the `.gitignore`. The command in the readme should force push those files in any case.